.semver

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
The version info is again stored in the `.semver` file. I realized it was a YAML file, so it's easy to work with it.